platforms/axplat-aarch64-bsta1000b/src/mem.rs

```diff
@@ -41,4 +41,14 @@ impl MemIf for MemIfImpl {
     fn mmio_ranges() -> &'static [RawRange] {
         &MMIO_RANGES
     }
+
+    /// Translates a physical address to a virtual address.
+    fn phys_to_virt(paddr: PhysAddr) -> VirtAddr {
```

Comment:
Put the implementation of `phys_to_virt`/`virt_to_phys` here, and remove the two functinos outside.